FOSDEM blogpost (#649)
unitaryfund/unitary.fund: src/content/blog/2025_fosdem.md

---
title: Quantum Computing at FOSDEM 2025
author: Purva Thakre
day: 13
month: 1
year: 2025
tags: 
  - Community
---

**After five years, the Quantum Computing devroom returns to FOSDEM.**

FOSDEM (Free and Open Source Software Developers' European Meeting) is a conference scheduled for 1-2 February 2025 at the ULB (Université libre de Bruxelles) Solboch Campus in Brussels, Belgium. It's a free event, and if you are in the area, you don't want to miss it! 

> The Quantum Computing devroom is slotted for the second half of Sunday 2 February 2025. We will be in room [K.4.401](https://fosdem.org/2025/schedule/room/k4401/).

Reviewer: ```suggestion
For the first time since 2020, FOSDEM will feature a Quantum Computing devroom, which is slotted for the second half of Sunday 2 February 2025. We will be in room [K.4.401](https://fosdem.org/2025/schedule/room/k4401/).
```

Accepted talks will focus on:
- Implementing a programming language for an analog quantum computer
- Successes and struggles of quantum software engineering
- Translating quantum concepts to software solutions
- Utilizing quantum programming languages to resolve disparities between quantum abstractions and software development
- Securing communications through qubits
- Vendor-neutral Python library for quantum circuits and algorithms which can be transpiled to IBM’s Qiskit, Google Cirq, AWS Braket, or to extend the library to run on other hardware.
- Platform agnostic quantum open-source tools
- Open source library for constructing verifiable measurement-based quantum computation (MBQC) patterns
- Cross-platform quantum compiler designed for frontend-agnostic, high-performance compilation of quantum circuits

We hope to see you in Brussels!

## Useful Links

- [FOSDEM 2025 website](https://fosdem.org/2025/)
- [FOSDEM Practical Information including transporation](https://fosdem.org/2025/practical/)
- [Quantum Computing devroom](https://fosdem.org/2025/schedule/track/quantum/)
- [FOSDEM Code of Conduct](https://fosdem.org/2025/practical/conduct/)

> - [Unitary Fund](https://unitary.fund/)

Author: Should this be Fund or Foundation? I see the switch from one to the other is currently WIP.

Reviewer: Foundation

---
title: Quantum Computing at FOSDEM 2025
author: Purva Thakre
day: 10
month: 1
year: 2025
tags: 
  - Community
---

**After five years, the Quantum Computing devroom returns to FOSDEM.**

> FOSDEM (Free and Open Source Software Developers' European Meeting) is a conference scheduled for 1-2 February 2025 at the ULB (Université libre de Bruxelles) Solboch Campus in Brussels, Belgium. The Quantum Computing devroom is slated for the second half of 2 February 2025.

Reviewer: ```suggestion
FOSDEM (Free and Open Source Software Developers' European Meeting) is a conference scheduled for 1-2 February 2025 at the ULB (Université libre de Bruxelles) Solboch Campus in Brussels, Belgium. It's a free event, and if you are in the area, you don't want to miss it! 

The Quantum Computing devroom is slotted for the second half of Sunday 2 February 2025. We will be in room [K.4.401](https://fosdem.org/2025/schedule/room/k4401/).
```

<p align="center">
  <a href="https://fosdem.org/2025/schedule/"> 
    <img src="/images/fosdem_25.png" />
  </a>
</p>

FOSDEM serves as a venue for open-source developers around the world to meet and facilitate conversations about the open-source ecosystem. This year, we hope to foster connections between the classical and quantum open-source ecosystems. Talks and presentations in each devroom (developer room) are suited to a particular topic of interest. In late 2024, we [put out a call](https://unitary.fund/community/FOSDEM_2025/) for the quantum devroom on talks related to: quantum software, quantum compilation, quantum programming languages, securing quantum communications, etc. 

> 

Reviewer: I propose to remove this section all together. This blog post is more a call for attendance, we want to be short and direct. The call for participation was a story for last year :)

> The full schedule is available [here](https://fosdem.org/2025/schedule/track/quantum/). People with any or zero familiarity with quantum computing are welcome! After the event, recorded talks will be available at the same link.

Reviewer: nice note! We can add some more
```suggestion
The full schedule is available [here](https://fosdem.org/2025/schedule/track/quantum/). People with any or zero familiarity with quantum computing are welcome! If you are at FOSDEM for a "classical" track on Saturday and you are curious to know more about Quantum Computing, join our devroom on Sunday! After the event, recorded talks will be available at the same link.
```

Accepted talks will focus on:
- Implementing a programming language for an analog quantum computer
- Successes and struggles of quantum software engineering
- Translating quantum concepts to software solutions
- Utilizing quantum programming languages to resolve disparities between quantum abstractions and software development
- Securing communications through qubits
- Vendor-neutral Python library for quantum circuits and algorithms which can be transpiled to IBM’s Qiskit, Google Cirq, AWS Braket, or to extend the library to run on other hardware.
- Platform agnostic quantum open-source tools
- Open source library for constructing verifiable measurement-based quantum computation (MBQC) patterns

> - Cross-platform quantum compiler designed for frontend-agnostic, high-performance compilation of quantum circuits

Reviewer: This feels a bit redundant given that we have titles and abstracts in the schedule. I say we remove it in the interest, once again, to make this post short and sweet

We hope to see you in Brussels!

## Useful Links

- [FOSDEM 2025 website](https://fosdem.org/2025/)
- [FOSDEM Practical Information including transporation](https://fosdem.org/2025/practical/)
- [Quantum Computing devroom](https://fosdem.org/2025/schedule/track/quantum/)
- [FOSDEM Code of Conduct](https://fosdem.org/2025/practical/conduct/)
- [Unitary Foundation](https://unitary.foundation/)

## Organizers

> If you have a specific request or question, you can reach out directly to the organizers:
> 
> - quantum-devroom-manager@fosdem.org
> - Alessandro Cosentino (cosenal [at] gmail.com)
> - Nate Stemen (nate [at] unitary.foundation)
> - Purva Thakre (purva.athakre [at] gmail.com)

Reviewer: ```suggestion
If you have a specific request or question, you can reach out directly to the organizers at quantum-devroom-manager@fosdem.org
```
